client/src/components/Project/Scrums/index.jsx

```diff
@@ -266,7 +250,7 @@ class Scrum extends Component {
       render: (value, record) => {
         return (
           <Editable
-            html={value}
+            html={!value ? '' : value}
```

Comment:
@IsraaSulaiman If you want to save a few keystrokes, you can usually do `html={value || ''}`